data_types/softlayer_load_balancer.go

```diff
@@ -0,0 +1,25 @@
+package data_types
+
+type SoftLayer_Load_Balancer_Update_Parameters struct {
+	Parameters []SoftLayer_Load_Balancer_Update `json:"parameters"`
+}
+
+type SoftLayer_Load_Balancer_Update struct {
+	SecurityCertificateId *int `json:"securityCertificateId"`
+}
+
+type SoftLayer_Load_Balancer struct {
```

Comment:
@tmulhern3 I think having the datatype names themselves be consistent with SoftLayer would be the priority.  Having the file names resemble the contained data types may be less important, since ultimately a package is mostly managed as a single unit anyway, irrespective of the actual number/names of files it is separated into.  At least that's been more or less my observation when looking at other Go code.  Thoughts?